packages/neos-ui-redux-store/src/UI/EditModePanel/index.js

```diff
@@ -30,11 +30,9 @@ export const actions = {
 // Export the reducer
 //
 export const reducer = handleActions({
-    [system.INIT]: () => $set(
+    [system.INIT]: payload => $set(
         'ui.editModePanel',
-        new Map({
-            isHidden: true
```

Comment:
I think we should do either or. It's really hard to trace the source of a particular part of state now, good source for bugs.